packages/npm-dependencies-validation/test/utils/packageJsonUtil/packageJsonUtil.spec.ts

```diff
@@ -29,4 +30,18 @@ describe("packageJsonUtil unit test", () => {
     const res = await isCurrentlySupported(packageJsonPath);
     expect(res).to.be.false;
   });
+
+  it("manged by npm", async () => {
+    const packageJsonPath =
+      "test/utils/packageJsonUtil/projects/npm-managed/package.json";
+    const res = await isCurrentlySupported(packageJsonPath);
+    expect(res).to.be.true;
+  });
+
+  it("non existing package.json", async () => {
+    const packageJsonPath =
+      "test/utils/packageJsonUtil/projects/not-existing-folder/package.json";
+    const res = await isCurrentlySupported(packageJsonPath);
```

Comment:
none existing path "is supported" because we only define conditions for what is NOT supported?

Reply:
I have added another test that checks this flow